build.sh

```diff
@@ -5,10 +5,18 @@ set -euxo pipefail
 OS_VERSION="$1"; shift
 ARCHITECTURE="$1"; shift
 
-packer build \
-  -var os_version="$OS_VERSION" \
-  -var-file "var_files/common.pkrvars.hcl" \
-  -var-file "var_files/$ARCHITECTURE.pkrvars.hcl" \
-  -var-file "var_files/$OS_VERSION/$ARCHITECTURE.pkrvars.hcl" \
-  "$@" \
-  openbsd.pkr.hcl
+flags=(
+  "-var os_version=$OS_VERSION"
+  "-var-file var_files/common.pkrvars.hcl"
+  "-var-file var_files/$ARCHITECTURE.pkrvars.hcl"
```

Comment:
This does not seem to work. This causes `-var os_version` to be passed as a single flag, when it should be passed as two. See: https://github.com/cross-platform-actions/openbsd-builder/actions/runs/10540382887/job/29205708401?pr=19#step:8:64.

Reply:
This is probably a quoting issue. I'll investigate.

Reply:
https://github.com/cross-platform-actions/openbsd-builder/pull/19/commits/cd849cdddc0abdc7dcfed16e169812034ceee9ae